client/reducers/conditionReducer.js

```diff
@@ -13,6 +13,9 @@ const conditionReducer = handleActions(
       const { latitude, longitude } = action.payload;
       return { ...state, latitude, longitude };
     },
+    [actionTypes.SET_PRICE_RANGE](state, action) {
```

Comment:
if you know price_range should be placed within conditionreducer, then why the price component is outside condition acting standalone?

Reply:
I need to separated it to organize set of functionality. The previous code in price is merge in condition. I think its better to separate it . Easy to debugged.